client/proposer-metrics/src/lib.rs

```diff
@@ -82,6 +85,26 @@ impl Metrics {
 				))?,
 				registry,
 			)?,
+			hit_block_size_limit: register(
+				CounterVec::new(
```

Comment:
Why do we need `CounterVec` here? We can just use a `Counter` and only increment when the limits are hit. When debugging we can look at graphs and see when this metric increments and then look at logs.

Reply:
Ahh can we than just do `rate()` and see the increase?

Reply:
Yes I think that would work for comparing different periods where the rate of full blocks would increase more.

Reply:
You could go for `CounterVec` with `overweight` and `oversized` labels and call the metric `substrate_proposer_block_error_count`. The Vec allows you to count events separately per label. 
One advantage of using counter vec would be that in Grafana you can group by the label and just use a single query instead of 2 in this case, but it really doesn't matter, both work fine.